CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Internal: Remove `image_quality` breakdowns from `sdk_warnings` response because the field will be soon deprecated.
 - Internal: Moved `geckodriver` from `dependencies` to `devDependencies`.
 - Internal: Added ability to run tests without the use of the mock server for the UI tests.
+- UI: Accessibility - Change Cross Device Send Link alternate options text an ARIA heading level 2
```

Comment:
I'm not sure I get this sentence.

Maybe:

```suggestion
- UI: Accessibility - Change Cross Device Send Link alternate option text to be an ARIA heading level 2
```

Reply:
Basically trying to say that this text is a ARIA heading level 2. In the code it's referred to in the copy keys as `get_link.link_divider`.
<img width="534" alt="Screenshot 2021-07-21 at 12 20 08" src="https://user-images.githubusercontent.com/2497081/126480891-dd752268-05ec-4815-a81c-c0ac613d92c1.png">